handbook/people-ops/spending-company-money.md

```diff
@@ -1,3 +1,67 @@
 # Spending company money
+Please see below for a list of reimbursable expenses (all in USD).
+
+## One-time expenses
+**Laptop (engineers):** $3500
+
+**Laptop (non-engineers):** $2000
+
+**Laptop coverage:** $380
+
+**Monitor:** $380
+
+**Headphones:** $200
+
+**Keyboard:** $130
+
+**Mouse or trackpad:** $80 or $130
+
+**USB-C Adapter:** $70
+
+**HDMI/monitor cable:** $15
+
+**Laptop stand:** $40
+
+**Desk:** $500
+
+**Ergonomic chair:** $200
+
+
+
+## Ongoing expenses
+**Software:** case by case, please get your manager’s approval.
+
+**Internet/WiFi plan:** $60/month
+
+**Cell phone plan:** $75/month (no need to expense, this is automatically added onto your pay).
+
+**Coworking/office space:** TBD
```

Comment:
I have some benchmarks for this one, but I'm actually leaning towards using a "average cost" approach here, rather than a maximum... I don't want to penalize people in SF/NYC/Tokyo/London vs. other locations, so I sort of like the "here's an average, we trust you to find the right deal" approach. What's your take?

Reply:
That sounds fair @dadlerj.

Reply:
Agreed!